apsuite/optics_analysis/optics_correction.py

```diff
@@ -0,0 +1,220 @@
+"""."""
+
+from copy import deepcopy as _dcopy
+from collections import namedtuple as _namedtuple
+import numpy as np
+
+import pyaccel
+from pymodels import bo, si
+
+
+class OpticsCorr():
+    """."""
+
+    METHODS = _namedtuple('Methods', ['Additional', 'Proportional'])(0, 1)
+    CORR_STATUS = _namedtuple('CorrStatus', ['Fail', 'Sucess'])(0, 1)
+
+    def __init__(self, model, acc, dim='4d', knobs_list=None,
+                 method=None, correction_method='loco'):
+        """."""
+        self.model = model
+        self.acc = acc
+        self.dim = dim
+        self.corr_method = correction_method
```

Comment:
treat it as a hidden attribute with a property, just like the `method` attribute.